blockchyp/blockchyp_client.py

```diff
@@ -39,6 +42,10 @@ def ping(self, terminal):
         """
         Tests connectivity with a terminal.
         """
+        route = self.__resolve_terminal_route(terminal)
+        reply = self.__terminal_post(route, "/test", payload={"terminalName": terminal})
+
```

Comment:
this pattern for assembling request payloads won't hold up as the messages get larger, just bear that in mind